internal/pkg/store/store.go

```diff
@@ -66,7 +66,7 @@ func New() (*Store, error) {
 		// See https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DNSLimitations.html#limits-service-quotas
 		// > To view limits and request higher limits for Route 53, you must change the Region to US East (N. Virginia).
 		// So we have to set the region to us-east-1 to be able to find out if a domain name exists in the account.
-		domainsClient: route53domains.New(sess, aws.NewConfig().WithRegion("us-east-1")),
+		route53Svc:    route53API.New(sess, aws.NewConfig().WithRegion("us-east-1")),
```

Comment:
Is this still valid? Do we want it to be in `us-east-1` or can we ditch the region now?

Reply:
`To view limits and request higher limits for Route 53, you must change the Region to US East (N. Virginia). To view limits and request higher limits for Resolver, change to the applicable Region.`
Yeah I think this applies to both route53 API and route53domain API.